src/main/java/com/hubspot/jinjava/el/ext/AstRangeBracket.java

```diff
@@ -1,21 +1,19 @@
 package com.hubspot.jinjava.el.ext;
 
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.Iterator;
-
-import javax.el.ELContext;
-import javax.el.ELException;
-import javax.el.PropertyNotFoundException;
-
 import com.hubspot.jinjava.objects.collections.PyList;
-
 import de.odysseus.el.misc.LocalMessages;
 import de.odysseus.el.tree.Bindings;
 import de.odysseus.el.tree.impl.ast.AstBracket;
 import de.odysseus.el.tree.impl.ast.AstNode;
 
+import javax.el.ELContext;
```

Comment:
@boulter 
Not sure which is right. Is the order correct?
1. java.*
2. javax.*
3. com.*
4. de.*